### DISCUSSION TITLE
Do I have Admin rights?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#64407] [New to GitHub] [answered]
Hi!

A real NOOB question!  I am trying to hook my Repos to the Streamlit Community Cloud.  I cannot make it work and have tried everything suggested by the SL community several times.  The only think I have not been able to confirm is whether I have Admin rights on my repo.  I have a free personal account and have set up all repos as Public.  How can I tell if I have admin rights, I have assumed this is how it works by default but am getting desperate!

S

**Comment by Kaisama** [comment, marked as answer]
No worries, we all start somewhere! To check if you have admin rights on your repository in GitHub, follow these steps:

1-Login to GitHub:
Make sure you are logged into your GitHub account.

2-Navigate to Repository:
Go to the repository that you're trying to connect to the Streamlit Community Cloud.

3-Repository Settings:
Once you're in your repository, click on the "Settings" tab located near the top-right corner of the repository page.

4-Options on the Left Sidebar:
In the left sidebar, click on "Options."

5-Manage Access:
Under "Options," there should be a section called "Manage access." Click on it.

6-Permissions:
On the "Manage access" page, you will see a list of users and their corresponding roles. If you have admin rights, you should see your username listed with the "Admin" role next to it.

If you see the "Admin" role next to your username, then you have admin rights on the repository. If not, you might have a different role such as "Collaborator" or "Member," which might not have the necessary permissions to connect your repository to the Streamlit Community Cloud.

If you don't have admin rights, you might need to contact the repository owner or someone with admin privileges to grant you the necessary permissions. Additionally, if you are the owner of the repository and you still don't see the "Admin" role next to your username, there might be some configuration issue or a problem that needs to be resolved.

Remember that GitHub's interface might evolve over time, so if you're having trouble finding certain options, refer to GitHub's official documentation or community for the most up-to-date information.

**Reply by ghost** [reply]
Thank you so much for answering my question.  However when I get to step 4, there is no 'Options' in the left sidebar and no where obvious to manage access.... thank you so much.  Screen shot enclosed.
![2023-08-20-163431_2560x1440_scrot](https://github.com/community/community/assets/134408604/a6ac79dc-ea8a-4d0c-9b48-993a890f468d)

**Comment by shubhamArkenea** [comment]
On the step 4, no 'Options' in the left sidebar, what will be the next steps to check permissions?

**Reply by Andante71** [reply]
If you see "collaborators n teams" up top in the left side there is a manage access section for members with their roles

**Reply by anubiaodoosh** [reply]
In the screenshot posted by **@ghost**, you can see the option **Collaborators**, instead of the common **Collaborators and teams**. I am experiencing the same situation with a specific project. When you click on **Collaborators** you will see the collaborators of the project but not their roles, so you cannot change them.

Do you know when does this happen? Recently, an user transferred the ownership of his repository to me, but I can't modify the collaborator roles, I can't even see them.
